actions/chce_usera.sh

```diff
@@ -2,15 +2,55 @@
 # Tworzenie nowego uzytkownika, z dostepem do sudo i kopia authorized_keys
 # Autor: Radoslaw Karasinski
 
-read -p "Podaj nazwe nowego uzytkownika: " username
+# if no sudo, then exit
+if [ "$(id -u)" != "0" ]; then
+	echo "Musisz uruchomić ten skrypt jako root" 1>&2
+	echo "Spróbuj sudo $0"
+	exit 1
+fi
+
+# Ask for the new user name
+if [ $# -eq 0 ]
+then
+	read -p "Podaj nazwe nowego uzytkownika: " username
+else
+	username=$(echo $1)
+fi
 
+# exit if the user name is empty
 if sudo id "$username" &>/dev/null; then
     echo "Podany uzytkownik juz istnieje!"
     exit
 fi
 
+read -p "Czy chcesz podać hasło? Jeśli nie zostanie ono wygenerowane [T/n] " c
```

Comment:
Nie lepiej zrobic, podaj hasalo (puste jeżeli nie), cos jak przy generowaniu kluczy SSH?

Reply:
Wsm racja, po prawię to później :) Tylko nwm jak to nałożyć na ten PR